compiler/src/dotty/tools/dotc/reporting/Message.scala

```diff
@@ -176,8 +175,8 @@ object Message:
       def needsExplanation(entry: Recorded) = entry match {
         case param: TypeParamRef => ctx.typerState.constraint.contains(param)
         case param: ParamRef     => false
-        case skolem: SkolemType => true
-        case sym: Symbol =>
+        case skolem: SkolemType  => true
+        case sym: Symbol         =>
```

Comment:
```suggestion
        case sym: Symbol =>
```
There is nothing on the right of `=>` to align.

Reply:
Thanks, I go back & forth on this style. The question is, in mix of one-liners, is the odd arrow also aligned?

Also default case: `case _ => ???` where it's always ugly to align.

I'm not a huge fan of formatters, but for this use case, it would restore my decision budget.